# TimestampDebug.msg
Header header
std_msgs/Time timestamp

int32 counter

# std_msgs/Time ap20_timestamp
# float64 node_arrival_delta
# float64 ap20_msg_deltaap20_timestamp